src/main/scala/uk/gov/nationalarchives/consignmentexport/BagMetadata.scala

```diff
@@ -18,7 +18,14 @@ class BagMetadata(keycloakClient: KeycloakClient)(implicit val logger: SelfAware
     private def isStringNullOrEmpty(s: String): Boolean = s == null || s.trim.isEmpty
 
     def isUserRepresentationComplete: Boolean = {
```

Comment:
I was wondering, should/could `isUserRepresentationComplete` be made private now?

Reply:
Good point